src/main/java/io/appform/dropwizard/actors/actor/BaseActor.java

```diff
@@ -99,8 +99,19 @@ protected BaseActor(
                 this::isExceptionIgnorable);
     }
 
+    /*
+        Override this method in your code for custom implementation.
+     */
+    protected boolean handle(Message message, MessageMetadata messageMetadata) {
+        return handle(message);
+    }
 
-    abstract protected boolean handle(Message message) throws Exception;
+    /*
+        Override this method in your code for custom implementation.
+     */
+    protected boolean handle(Message message) {
```

Comment:
Mark this as Deprecated as we shall remove this on 2x line

Reply:
added.